File: backend/Core/Models/ContactModel.cs
namespace Core.Models
{
    public class ContactModel
    {
        public int ContactId { get; set; }

        public int UserId { get; set; }

        public decimal Name { get; set; }

        public required string PhoneNumber { get; set; }        

        public DateTime SaveDate { get; set; }
    }
}

File: frontend/src/app/models/contact.model.ts
export interface IContactModel {
    contactId: number;
    name: string;
    phoneNumber: string;
    saveDate: Date;
    userId: string;
  }
  
  export class ContactModel implements IContactModel {    
    public contactId: number = 0;
    public saveDate: Date = new Date();
  constructor(
    public name: string,
    public phoneNumber: string,    
    public userId: string,  
  ) { }
}